code/game/dna/dna2.dm

```diff
@@ -144,13 +144,13 @@ var/global/list/datum/dna/gene/dna_genes[0]
 	// FIXME:  Species-specific defaults pls
 	if(!character.h_style)
 		character.h_style = /decl/sprite_accessory/hair/bald
-	var/list/hair_types = subtypesof(/decl/sprite_accessory/hair)
+	var/list/hair_types = decls_repository.get_decls_of_subtype(/decl/sprite_accessory/hair)
```

Comment:
See above--this changed it from a path to an instance. Probably wanted `get_decl_paths_of_subtype()`.